js/script.js

```diff
@@ -0,0 +1,6 @@
+let target = document.querySelectorAll('.favorite-icon')
+
+target.forEach(cat => {
+    cat.addEventListener('click', function () {
+        this.parentElement.submit()
+    })})
```

Comment:
COOL. This means it's actually not that hard to make it a single form, if you apply the favourite icon class to a radio button or something